spec/controllers/submissions_controller_spec.rb

```diff
@@ -854,13 +854,45 @@
       end
 
       describe '#manually_collect_and_begin_grading' do
-        before do
+        it 'it should respond with 302' do
```

Comment:
delete the "it" inside the string

Reply:
whoops sorry about that, thought I got that one!